tensorflow/core/util/tensor_format.h

```diff
@@ -145,6 +146,7 @@ inline int GetTensorDimsFromSpatialDims(int num_spatial_dims,
     case FORMAT_NHWC_VECT_W:
       return num_spatial_dims + 3;  // Include N,C,VectDim.
   }
+  return -1;
```

Comment:
Instead of the return statement, could you please add it as a default condition to the switch with a log statement. This is the pattern followed in the rest of the file.

```
default:
      LOG(FATAL) << "Unknown format " << format;
      return -1;  // Avoid compiler warning about missing return value
```

Reply:
fixed